Assets/Scripts/Player/SoundAlerter.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SoundAlerter : MonoBehaviour
+{
+    //public delegate void OnSoundEmit(float volume, Vector3 source);
+    //public static event OnSoundEmit OnSoundEmitEvent;
```

Comment:
In future, please remove unused code that has been commented out unless there is a good reason for it to stay.